packages/web-components/fast-element/src/templating/html-directive.ts

```diff
@@ -179,7 +179,11 @@ export abstract class Binding<TSource = any, TReturn = any, TParent = any> {
     /**
      * Evaluates the binding expression.
      */
-    evaluate: Expression<TSource, TReturn, TParent>;
+    evaluate: Expression<TSource, TReturn, TParent> = {} as Expression<
```

Comment:
For this one I really wasn't sure which approach was best. I could probably have gone with this too
```suggestion
    evaluate!: Expression<TSource, TReturn, TParent>;
```